CMakeLists.txt

```diff
@@ -73,12 +73,12 @@ add_custom_target(run
 # out forcing re-builds when they might not want them.
 add_custom_target(setup-and-run
     COMMAND ${CMAKE_MAKE_PROGRAM} install
-    COMMAND ${CMAKE_MAKE_PROGRAM} image
+    COMMAND ${CMAKE_MAKE_PROGRAM} disk-image
     COMMAND ${CMAKE_MAKE_PROGRAM} run
     USES_TERMINAL
 )
 
-add_custom_target(image
+add_custom_target(disk-image
```

Comment:
Actually, perhaps we can just drop this in favor of invoking `qemu-image` directly? Seems much more clear naming-wise, too.

Reply:
I think we have this as abstraction point for other image types (?)

Reply:
I don't know the exact history, but I would not be surprised `image` used to be the only option and was retained for compatibility.